src/components/EventOverview/EventsPage.tsx

```diff
@@ -149,7 +155,9 @@ const EventsPage = () => {
                 date={event.eventDate}
                 eventTitle={event.name}
                 isActive={event.isActive}
-                isNew={event.id === addedEventId}
+                isNew={
+                  event.id === addedEventId || event.id === archivedEventId
```

Comment:
there is a scenario where if you add a new event and then immediately archive a **different** event, you get 2 cards that are highlighted at once - is it a big deal to get it so that only the last event that changed gets highlighted?